packages/create-bison-app/template/prisma/schema.prisma

```diff
@@ -12,32 +12,32 @@ datasource db {
 }
 
 model Profile {
-  id        String   @id @default(cuid())
-  userId    String   @unique
+  id        String   @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  userId    String   @unique @db.Uuid
   firstName String
   lastName  String
   image     String?
   createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
+  updatedAt DateTime @default(now()) @updatedAt
```

Comment:
this does give the correct default if someone were to insert via the database directly, but it's not going to keep the timestamp should they update the record. It's certainly not hurting anything since this value will always be provided by Prisma, but just wanted to point that out.

Reply:
This solves the issue of needed a generated timestamp on create
Currently from the DB -- this throws a `updatedAt` required. 

On initial create -- we get the current timestamp for both created and updated (expected) 
On update -- updated has the correct value.